src/commands/dev/index.js

```diff
@@ -97,6 +99,17 @@ function initializeProxy(port) {
     }
   })
 
+  const headersFiles = Array.from(new Set([
+    path.resolve(projectDir, '_headers'),
+    path.resolve(distDir, '_headers'),
+  ]))
+
+  let headerRules = {}
+  onChanges(headersFiles, () => {
+    console.log(`${NETLIFYDEVLOG} Reloading headers files`, headersFiles.map(p => path.relative(projectDir, p)))
+    headerRules = headersFiles.reduce((prev, curr) => Object.assign(prev, parseHeadersFile(curr)), {})
```

Comment:
Should we catch `parseHeadersFile` errors in this callback and log them?

Reply:
We're throwing those errors without catching because we want to halt the execution if a rule rule is not correct.